src/theme/SearchBar/styles.module.css

```diff
@@ -11,6 +11,16 @@
     stroke-width: 2;
 }
 
+.hotkey {
+    font-size: 12px;
+    border: 2px solid var(--ifm-color-gray-700);
+    padding: 3px;
+    border-radius: 8px;
+    color: var(--ifm-color-gray-600);
+    margin-left: 3px;
+    font-weight: bold;
+}
+
```

Comment:
Consider brightening the indicator on hover.

Reply:
I like the way it is. I will let that up to the judge.

Reply:
+1 on brightening the indicator on hover since the search icon highlights on hover. Ideally we should have a search bar like this but hover for now works. 

<img width="481" alt="Screenshot 2024-01-04 at 09 39 24" src="https://github.com/surrealdb/docs.surrealdb.com/assets/35943047/ee78b262-9efe-4ba4-a7df-80e323388997">

Reply:
Done.